synapse/daemon.py

```diff
@@ -286,6 +302,7 @@ def _loadDmonCells(self):
             self.loadDmonCell(name)
 
     def loadDmonCell(self, name):
+        import synapse.cells as s_cells
```

Comment:
why the deferred import here?

Reply:
There's an import cycle.

Reply:
Looking quick - it seems like the import cycle was on the s_dmon.Share class. Can we move that class to his own file? `synapse.lib.share` ?

Reply:
done